### PR TITLE
Javadoc

### PR DESCRIPTION
**Pull request by rsavin**
Add javadocs and readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,158 @@
 # OPFMaps
+OPFMaps is an open source library which provides an easy way for developers to integrate android maps into their apps.
```
Comment (akarimova) [inline README.md:2]: ->different Android?

File: README.md [hunk 2]
```diff
@@ -1 +1,158 @@
...
+
+Currently OPFMaps supports the following maps: [Google Maps][google-maps], [Amazon Maps][amazon-maps], [Osmdroid][osmdroid], [Yandex Maps JavaScript API][yandex-web-maps].
+
+##How To Use
```
Comment (akarimova) [inline README.md:6]: to

File: README.md [hunk 3]
```diff
@@ -1 +1,158 @@
...
+*Yandex Maps Provider*:
+
+```gradle
+compile 'org.onepf:opfmaps-yandex-web:0.1.0@aar'
+```
+
+
+**Initialization**
+
+You must init `OPFMapsHelper` before using of OPFMaps. You must do it in the `Main Thread`. `Application.onCreate()` method is the best place for this.
```
Comment (akarimova) [inline README.md:66]: of?

File: opfmaps-providers/google/src/main/java/org/onepf/maps/google/GoogleMapProvider.java [hunk 4]
```diff
@@ -26,6 +26,8 @@
 import org.onepf.opfutils.OPFLog;
 
 /**
+ * Google Map Provider.
+ *
  * @author Anastasiia Karimova
```
Comment (akarimova) [inline opfmaps-providers/google/src/main/java/org/onepf/maps/google/GoogleMapProvider.java:31]: Please replace every AK with Roman Savin

File: README.md (hunk 2 above)
Comment (rsavin) [inline README.md:6]: ?

**Pull request merged**